checks.d/spark.py

```diff
@@ -624,6 +648,7 @@ def _rest_request(self, address, object_path, service_name, *args, **kwargs):
             url = urljoin(url, '?' + query)
 
         try:
+            self.log.debug('Spark check URL: %s' % url)
```

Comment:
Do we really need this line? I can see it being of value when debugging support cases, but it'll be quite verbose, no? Your call.